Calendr/Components/Label.swift

```diff
@@ -65,7 +75,7 @@ class Label: NSTextField {
         setContentHuggingPriority(.fittingSizeCompression, for: .horizontal)
     }
 
-    private func setUpBindings() {
+    fileprivate func setUpBindings() {
```

Comment:
These changes to `fileprivate` are probably not necessary since you created a separate component, right?

Reply:
Yep, forgot to change back to private